coral/src/app/features/approvals/utils/request-status-helper.ts

```diff
@@ -1,9 +1,16 @@
 import { RequestStatus } from "src/domain/requests";
 import { ChipStatus } from "@aivenio/aquarium";
 
-const requestStatusChipStatusMap: {
-  [key in RequestStatus]: ChipStatus;
-} = {
+// @TODO add safe list
```

Comment:
What is the safe list? :o

Reply:
I thought we may have / find a way to either have one typesafe list of requests where we can derive all other types / values from, but I could not find a good solution (in a timeboxed time frame). So I left so comment to remind me of it later.

Reply:
>   one typesafe list of requests where we can derive all other types / values from

Sorry, I don't understand ^^ Could you provide an example of what you mean? The `statusList` seems already typesafe to me?

Reply:
You can remove any value in there and it's still valid. I would like something where TS complains if a value is missing!

Reply:
Ah of course. I don't believe that kind of exhaustivity is possible with array (yet, I think tuples and stuff are coming up).

closests we could get is something like this probably :o https://stackoverflow.com/a/55266531

Reply:
Will edit!

There may be a solution for that problem out there but I thought it's a bit unreasonable for this use case to spent too much time on it! (I was about to go down the rabbit hole to see why the `enum` from the openapi def is not a `enum` here but stopped myself 🐰 )

Will make a better comment!